.concourse/tasks/upgrade.sh

```diff
@@ -1,10 +1,40 @@
 #!/bin/bash
 
+# Notes
+# - Parameters:
+#   - GKE_KEY
+#   - GKE_PROJECT
+#   - GKE_ZONE
+#
+# - Variables used by catapult:
+#   - GKE_CLUSTER_NAME
+#   - GKE_CLUSTER_ZONE
+#   - GKE_CRED_JSON
+#   - GKE_PROJECT
+#
+#   - BACKEND
+#   - CONFIG_OVERRIDE
+#   - ENABLE_EIRINI
+#   - KUBECFG
+#   - QUIET_OUTPUT
+#   - SCF_CHART
+#   - SCF_OPERATOR
+#
+# Variables used by kubectl, gcloud, ...
+#   - KUBECONFIG
+
 set -e
+set -x
 
 echo "Running the upgrade test"
 
-KUBECF_LATEST_RELEASE=$(cat kubecf-github-release/version)
+# Fail early for missing parameters
+GKE_KEY="$GKE_KEY"
```

Comment:
You don't have `set -o nounset` set, how does this fail?
Also, please use something like `: ${GKE_KEY:?}` instead.

Reply:
Oops. Ok.

Reply:
Done with commit 54507d8b51b33626385ae6769c4abdc6b8edcd11